src/main/java/org/bunge/bgginfo/scheduled/ScheduledTasks.java

```diff
@@ -1,27 +1,19 @@
 package org.bunge.bgginfo.scheduled;
 
-import java.util.Random;
-
 import org.bunge.bgginfo.bggintegration.service.BGGService;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
 
 @Component
 public class ScheduledTasks {
+    @Autowired
+    private BGGService bggService;
 
-    @Autowired 
-    BGGService bggService;
-
-	private static final Logger log = LoggerFactory.getLogger(ScheduledTasks.class);
- 
-    @Scheduled(fixedRate = 3000)
+    @Scheduled(fixedRate = 30000)
```

Comment:
As far as I can tell there's no need to slow the rate down like this. I haven't had any calls fails at 3000ms.

Reply:
Please change it back for approval.